erdjs/src/networkParams.ts

```diff
@@ -138,6 +138,30 @@ export class TransactionVersion {
     }
 }
 
+export class TransactionOptions {
```

Comment:
Not sure - perhaps on this class we should add construction utilities such as `TransactionOptions.fromFlags({ signHashInsteadOfContent: true, fooBar: false })` - not necessarily this way, and not necessarily now - but currently, the construction from a raw number (one might ask _what is about this number?_) isn't very meaningful.

Does this make sense?

Reply:
Yes, will add also a private constructor (`.fromFlags`)

Reply:
Done. created 'static' constructors such as `withDefaultVersion` or `withTxHashSigningOptions` for both TransactionVersion and TransactionOptions